internal/object.h

```diff
@@ -10,6 +10,11 @@
  * @brief      Internal header for Object.
  */
 #include "ruby/ruby.h"          /* for VALUE */
+#include "internal/class.h"     /* for RCLASS_IV_INDEX_TBL */
+
+#ifdef ROBJECT_IV_INDEX_TBL
+# undef ROBJECT_IV_INDEX_TBL
+#endif
```

Comment:
Just for curiosity, is `#ifdef` needed?
I've thought that C89 said that `#undef` silently does nothing if the macro is not defined.

Reply:
The `#ifdef` is not mandatory.  Just my style.

Reply:
(Actually I thought it was the project's style.  However after `git blame`-ing around for a while, I learned that I am the only one who is guarding `#undef`s.  So this is my style, but I have no strong reasoning to do so.  It is okay for me not to guard them).